.gitignore

```diff
@@ -1,3 +1,3 @@
 /target
-
-/out
\ No newline at end of file
+/out
+*_diff.png
```

Comment:
Is this necessary?

Reply:
I was copying all the images from the `out` folder to the `tests` folder, so I had this to make sure I didn't commit anything I shouldn't. I've reverted the change.